src/components/SelectionList/BaseSelectionList.js

```diff
@@ -372,8 +373,13 @@ function BaseSelectionList({
             return;
         }
 
+        // scroll is unnecessary if multiple options cannot be selected
+        if (!canSelectMultiple) {
+            return;
+        }
```

Comment:
I was thinking we add the `shouldScrollToTopOnSelect` so we don't have to create this if and actually use the new property.

I guess we either remove the new property or use it instead of creating the if, what do you think @eVoloshchak?

Reply:
I think it makes sense, let's remove it then and sorry about the confusion.